download.go

```diff
@@ -15,29 +16,25 @@ func extractArchive(path string) error {
 	return command.Run([]string{"tar", "zxvf", path})
 }
 
-func download(url string, logName string) error {
-	args := []string{"wget", url}
-	if command.VerboseEnabled {
-		return command.Run(args)
+func downloadForBuilder(b *builder.Builder) error {
+	res, err := http.Get(b.DownloadURL())
```

Comment:
As `http.Get()` uses `http.DefaultClient`, timeout is not set. We should set timeout for http client. FYI, the default timeout of `wget` is 900 seconds.